ivy/functional/frontends/jax/func_wrapper.py

```diff
@@ -74,3 +87,50 @@ def new_fn(*args, **kwargs):
 
 def to_ivy_arrays_and_back(fn: Callable) -> Callable:
     return outputs_to_frontend_arrays(inputs_to_ivy_arrays(fn))
+
+
+def handle_nans(fn: Callable) -> Callable:
```

Comment:
a lot of duplication, we should define this once in `ivy/func_wrapper.py` and then import this function into each frontend `func_wrapper.py` file, they are identical I believe right?

Reply:
Sure, that makes sense. I did it because I followed the same pattern as `inputs_to_ivy_arrays` that was also declared in multiple places.

Neverthess, fixed as requested and now can be directly imported from the `ivy.func_wrapper`. I did not added the `from ivy.func_wrapper import handle_nans` on top of the files as it will lead to unused import errors. Let 's just allow to directly import it from `ivy.func_wrapper` for all files.